src/forgot-password/LoadableForgotPasswordPage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import loadable from '@loadable/component';
+import Spinner from '../common-components/Spinner';
+
+const LoadableComponent = loadable(() => import('./ForgotPasswordPage.jsx'), {
+  fallback: <Spinner />,
```

Comment:
I think instead of spinner we can use bar like github

Reply:
Since we are using the spinner in other places as well like during token validation etc, I guess it doesn’t make sense to implement a new loader here. That's a good suggestion ad we can consider changing it later on after the launch.